CHANGELOG.md

```diff
@@ -1,23 +1,34 @@
 # Unreleased
- 
+
+# 9.15.0 (Feature release)
+
 ## New features
 
-### Replace back link placeholder URLs with javascript
+### Replace back link placeholder URLs with JavaScript
+
+We've added JavaScript to make the [back link component](https://design-system.service.gov.uk/components/back-link/) take users to the previous page by default.
+
+Before, to get the back link to work, you had to manually update placeholder text.
+
+Do not use this approach for production scenarios, where you cannot rely on JavaScript always being available.
```

Comment:
Can we change to "Do not use the back link component in production"? The previous two sentences refer to two different things - using the back link component with javascript and using the back link component with manually updated placeholder text. So I think it would help to be explicitly clear.

Reply:
That's a great point, thanks Jon.

```suggestion
Do not use the back link component in production, where you cannot rely on JavaScript always being available.
```

Reply:
I think this implies the back link component should never be used in production - what we're really saying is that this approach (which relies on JavaScript with no fallback) shouldn't be used.

Reply:
@jonathanglassman @vanitabarrett What about something like this?

```suggestion
Do not use JavaScript for production scenarios, where you cannot rely on JavaScript always being available.
```

Reply:
This is tricky to word, I'm still not sure if it's a bit misleading because users *can* use JavaScript. In our guidance in the design system we say:

> Where possible, ensure it works even when JavaScript is not available.
> If this is not possible, you should hide the back link when JavaScript is not available.

Reply:
Maybe something like "Do not rely on JavaScript for production scenarios..." or "Do not rely on JavaScript-only implementations for production scenarios..." is clearer?

Reply:
"Implementations" is specific about what to use JavaScript for, which sounds like it could help users. What do you think, @jonathanglassman? 

```suggestion
Do not rely on JavaScript-only implementations for production scenarios, where you cannot rely on JavaScript always being available.
```

Reply:
How about "Do not use this new Javascript-reliant approach for production scenarios, where you cannot rely on JavaScript always being available."

Reply:
I think that sounds good. Is this wording ok, @vanitabarrett?

```suggestion
Do not use this JavaScript-reliant approach for production scenarios, where you cannot rely on JavaScript always being available.
```

Reply:
Sounds good, thanks both!

Reply:
Thanks @Vanita! Thanks @jonathanglassman!